content/en/docs/setup/install/helm/index.md

```diff
@@ -17,6 +17,12 @@ installing Istio via [Istioctl](/docs/setup/install/istioctl/) or the
 
 This feature is currently considered [alpha](/about/feature-stages/).
 
+{{< warning >}}
+Prior to Istio 1.9.0 installation using the Helm charts required the arguments
```

Comment:
Maybe:

Prior to Istio 1.9.0, installations using the Helm charts required hub and tag arguments:
 `--set global.hub="docker.io/istio"` and `--set global.tag="1.8.2"`. As of Istio
 1.9.0 these are is no longer required.

Reply:
Changed

Reply:
Added as a boilerplate too